src/rmath.jl

```diff
@@ -56,9 +56,15 @@ function _import_rmath(rname::Symbol, jname::Symbol, pargs)
         $pdf($(pdecls...), x::Float64) =
             ccall(($dfun, libRmath), Float64, $dtypes, x, $(pargs...), 0)
 
+        $pdf($(pdecls...), x::Int64) =
+            ccall(($dfun, libRmath), Float64, $dtypes, x, $(pargs...), 0)
+
         $logpdf($(pdecls...), x::Float64) =
             ccall(($dfun, libRmath), Float64, $dtypes, x, $(pargs...), 1)
 
+        $logpdf($(pdecls...), x::Int64) =
+            ccall(($dfun, libRmath), Float64, $dtypes, x, $(pargs...), 1)
+
```

Comment:
Rather than having separate signatures for `Float64` and `Int64`, you should just be able to do `x::Union{Float64,Int64}`.